elastalert/create_index.py

```diff
@@ -248,7 +248,15 @@ def main():
     # To avoid a race condition. TODO: replace this with a real check
     time.sleep(2)
 
-    if(elasticversion > 5):
+    if(elasticversion > 6):
+        params = {'include_type_name': 'true'}
```

Comment:
see https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html

Reply:
The include_type_name parameter in the index creation, index template, and mapping APIs will default to false. Setting the parameter at all will result in a deprecation warning.